shared/bash_remediation_functions/service_command.sh

```diff
@@ -46,12 +46,20 @@ else
   chkconfig_state="off"
 fi
 
+inode=$(ls -di / | cut -d ' ' -f1)
```

Comment:
can't we just immediately exit out at the  beginning of the function? would save a lot of logic